views/templates/Status/index.tpl

```diff
@@ -1,85 +1,11 @@
 <?php
-use oat\taoSystemStatus\model\Check\CheckInterface;
 use oat\tao\helpers\Template;
-
-$report = get_data('report');
-$link = get_data('support_portal_link');
-$service = get_data('service');
-$reportsByStatus = get_data('reports_by_status');
-
-$configurationReports = $reportsByStatus[__('TAO Configuration')];
-unset($reportsByStatus[__('TAO Configuration')]);
-
-$configurationValuesReports = $reportsByStatus[__('Configuration Values')];
-unset($reportsByStatus[__('Configuration Values')]);
-
-$healthReports = $reportsByStatus[__('Health/Readiness check')];
-unset($reportsByStatus[__('Health/Readiness check')]);
-
-$monitoringReports = $reportsByStatus[__('Monitoring / Statistics')];
-unset($reportsByStatus[__('Monitoring / Statistics')]);
-
 ?>
 
 <link rel="stylesheet" href="<?= Template::css('systemstatus.css') ?>" />
 
-<div class="data-container-wrapper flex-container-full" id="system-status-report">
-    <div class="grid-container">
-        <div class="grid-row">
-            <div class="col-10">
-                <h1>Welcome to TAO's status page hub</h1>
-            </div>
-            <div class="col-2">
-                <?php if ($link): ?>
-                <a href="<?= $link ?>" target="_blank" class="btn-info large support_portal_link"><span class="icon-save"></span> Visit Support Portal</a>
```

Comment:
We need that link to be shown on this page in case if `support_portal_link` option is configured in `SystemStatusService.conf.php`.
How we can pass it to frontend? Do you need an endpoint for that purpose?

Reply:
it is still possible to pass this link using PHP param